server/sonar-web/src/main/js/apps/marketplace/components/EditionsStatusNotif.tsx

```diff
@@ -49,7 +50,8 @@ export default class EditionsStatusNotif extends React.PureComponent<Props, Stat
   };
 
   renderStatusAlert() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The function renderStatusAlert has a cyclomatic complexity of 11 which is higher than the threshold of 10 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1541)